lib/classification_lifecycle.rb

```diff
@@ -14,6 +14,12 @@ def queue(action)
       raise ClassificationNotPersisted.new(message)
     end
     ClassificationWorker.perform_async(classification.id, action)
+    if classification.complete?
```

Comment:
My bad - completed here is referencing the state of the classification not the retirement state of subject counts. Ignore me.